src/feature/relay/dns.c

```diff
@@ -1559,7 +1568,9 @@ evdns_callback(int result, char type, int count, int ttl, void *addresses,
         log_notice(LD_EXIT, "More than half of our IPv6 requests seem to "
```

Comment:
Please increase the minimum number of DNS queries in these lines:
```
    if (n_ipv6_timeouts > 10 &&
        n_ipv6_timeouts > n_ipv6_requests_made / 2) {
```

But set it to 1 when TestingTorNetwork is 1.

Also, please replace the numbers by named constants. One of the reasons you changed the wrong code, is that there are multiple "10"s in this file.